City-of-Helsinki/helsinki-paatokset
# PP-295: Made frontend fixes in mobile view

Ticket: https://helsinkisolutionoffice.atlassian.net/browse/PP-295

**How to test**

Checkout branch and run `drush cr`

Go to some policymaker page and check 

- the margins between the mobile navigation and breadcrumbs in mobile
- margin below the tag in desktop and mobile
- h1 font size in desktop and mobile

Go to https://helsinki-paatokset.docker.so/fi/paattajat

Check

- margins below section titles inside the accordions
- in mobile view make sure that the policymaker titles don't overflow from the box and the arrow is vertically in the center of the box

Go to https://helsinki-paatokset.docker.so/fi/tietoa-paatoksenteosta (or create a page with that path)

Check the decision tree in mobile view that the titles don't overflow

Go to some case page that has a long tag i.ex. https://helsinki-paatokset.docker.so/fi/asia/hel-2022-000266
Check that the tag and its background color wraps nicely on smaller screens
